src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/BaseTest.cs

```diff
@@ -412,6 +412,37 @@ protected byte[] ReadAllBytesIgnoringLineEndings (Stream stream)
 			}
 		}
 
+		protected string GetPathToAapt2 ()
+		{
+
+			var exe = IsWindows ? "aapt2.exe" : "aapt2";
+			var path = Path.Combine (AndroidMSBuildDirectory, IsWindows ? "" : (IsMacOS ? "Darwin" : "Linux"));
+			if (File.Exists (Path.Combine (path, exe)))
+				return path;
+
+			path = Path.Combine (AndroidSdkPath, "build-tools");
+			foreach (var dir in Directory.GetDirectories (path, "*", SearchOption.TopDirectoryOnly).OrderByDescending (x => x)) {
```

Comment:
This is searching alphabetically, should it parse to a `System.Version` for the order-by?